code/game/turfs/open.dm

```diff
@@ -119,7 +117,7 @@
 	MakeSlippery(TURF_WET_PERMAFROST, 5)
 	return 1
```

Comment:
This whole proc is so bad its painful to watch
1 - make the effect proc universal for atoms
2 - move all checks inside their appropriate object procs
3 - loop once over things in contents and call the effect proc of those movables
4 - point at goof and laugh